channels/google/accounts.rst

```diff
@@ -47,9 +47,8 @@ Accounts
 Email Notification
    This option is **not available on Google channels**.
 
-   To configure the outgoing mail provider for system notifications,
-   use a :doc:`regular email notification channel </channels/email/accounts>`.
-   (Self-hosted installations only.)
+   (See the corresponding section on the
```

Comment:
I must confess that I'm not a big fan on removing essential information (again!).
Especially the "self hosted part only" is a crucial one.

I mean I do understand you want to keep it short, but we really have to stop removing important bits.

Reply:
I hear you. When I came (back) to this section, I just thought there was a lot of potential for confusion. We're basically saying:

> ### Setting that's not in the control panel
>
> This setting doesn't exist. But actually, it does, just only on regular email channels. So if you want to do this thing that maybe you weren't even thinking about doing, go there instead. But actually, you can only do that if you're self-hosted; on cloud-hosted instances, this setting _really_ doesn't exist. Like, at all.

and all that back-and-forth was just too much for me.

Importantly, I'm making an assumption about the questions in the reader's mind when they're on this page. My assumption is that they're thinking, "Hm, how do I set up a Microsoft 365 account?" and not "Hm, how do I configure my Microsoft 365 account to be the address for system notification emails?"

But as always, you know the customer (i.e. support caller) better than I do. I'll prepare an edit now and push that to this PR for your approval.